mautrix_facebook/portal.py

```diff
@@ -328,6 +328,18 @@ def _get_photo_id(url: Optional[str]) -> Optional[str]:
         except MatrixError:
             pass
 
+    @staticmethod
+    async def are_only_puppets_left(intent: IntentAPI, room_id: RoomID) -> bool:
+        try:
+            members = await intent.get_room_members(room_id)
+        except MatrixError:
+            members = []
+        for user_id in members:
+            puppet = p.Puppet.get_by_mxid(user_id, create=False)
```

Comment:
This could just use `get_id_from_mxid` instead of also getting the puppet from the cache or database.

Reply:
0aad804e79f1c457b3dedb538efb4b0bc091a4f6